### DISCUSSION TITLE
Azure Devops Integration for Qodana Cloud Docs

### DISCUSSION THREAD

**Discussion by alfkonee** [JetBrains/Qodana#141] [Ideas]
Hello There please would it be possible to include the integration steps for Azure Devops too in the [Qodana Cloud Forward Reports](https://www.jetbrains.com/help/qodana/cloud-forward-reports.html) Sections??

**Comment by tiulpin** [comment]
Hi! Thanks for the suggestion, we will update our docs on that soon. Also, all you need is to declare one variable in the job: the `QODANA_TOKEN` environment variable that you can get when you create a project on https://qodana.cloud/

**Reply by alfkonee** [reply]
Hello @tiulpin  I did that but for some reason the results aren't shipping to the cloud

**Reply by alfkonee** [reply]
Please Can we get some validation that this is working in Azure Devops

**Reply by tiulpin** [reply]
@alfkonee you can refer to the updated docs: https://github.com/JetBrains/Qodana/blob/2022.3/topics/qodana-azure-pipelines.md#qodana-cloud (will be published on our [web help](https://jetbrains.com/help/qodana) soon). Maybe you forgot to map the configured variable to the pipeline:

```yaml
  - task: QodanaScan@2022
     env:
       QODANA_TOKEN: $(QODANA_TOKEN)
```

Sorry for the long reply!
